magento/magento2
# #20413 admin__control-fields class should not add in Frontend ( in checkout page )

### Description (*)
fixed issue with admin__control-fields class on checkout

### Fixed Issues
1. https://github.com/magento/magento2/issues/20413: admin__control-fields class should not add in Frontend ( in checkout page )

### Manual testing scenarios (*)
1. Open frontend
2. Add any product in cart
3. Go to checkout and see shipping address form > Street Address Label
4. And inspect html here added admin__control-fields in fieldset
http://prntscr.com/m904ax

### Contribution checklist (*)
